src/parser/shared/modules/items/bfa/dungeons/RotcrustedVoodooDoll.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS/index';
+import ITEMS from 'common/ITEMS/index';
```

Comment:
Although not wrong, it would be more consistent with codebase to import it as `common/ITEMS`, omitting the `index`. Same with SPELLS.